### ISSUE TITLE
Transparency

### ISSUE THREAD

**Issue by maintux** [Automattic/Iris#42]
Hi, it's possible to add transparency support in color picker box?
Thanks

**Comment by monecchi** [comment]
Yep! Indeed it'd be great to be able to set alpha transparency to the selected colors.
